roles/install_nextcloud/tasks/nc_apps.yml

```diff
@@ -1,17 +1,10 @@
 ---
-- name: nc_apps | Parse the item values
+- name: "nc_apps | Parse the item values"
```

Comment:
The quotes here are not necessary since you are not templating anything.

Reply:
Actually you are making more inconsistent. We should **consistently** use quotes only when templating.

Reply:
Got it, I haven't realized this. Have changed it.